docs/spec.md

```diff
@@ -160,6 +160,21 @@ following constraints:
 * (C12) If `quantization_dimension` is not empty, then
   `0 <= quantization_dimension`.
 
+At the moment, `QuantizationScale` is a floating-point constant, but there is
+strong interest in integer-based scales, represented with multipliers and
+shifts. We are planning to explore this in the near future
+([#1404](https://github.com/openxla/stablehlo/issues/1404)).
+
+There is an ongoing discussion on the semantics of `QuantizationZeroPoint`,
+including the type, the values and whether there can be just one or
+potentially multiple zero points in a quantized tensor type. Based on the
+results of this discussion, the specification around zero points may change
+in the future ([#1405](https://github.com/openxla/stablehlo/issues/1405)).
```

Comment:
Should we also include #1406?

Reply:
Good point. I was hesitating when I originally wrote this up, but I think we should include a reference to this ticket indeed.